src/Microsoft.Win32.Primitives/src/System/ComponentModel/Win32Exception.Windows.cs

```diff
@@ -2,15 +2,10 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System;
-
 namespace System.ComponentModel
 {
-    public partial class Win32Exception : Exception
+    public partial class Win32Exception
```

Comment:
Ah. My small preference is for consistency (less likely someone would edit the code and remove it from the other one) but your call there.